src/components/YoutubeGrid/YoutubeGrid.component.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Card from '../YoutubeCard/YoutubeCard.component';
+import data from '../../mock/youtube-videos-mock.json';
+import { StyledGrid } from './YoutubeGrid.styles';
+
+function Grid() {
+  const videosInfo = data.items;
```

Comment:
In order to make this more reusable, I would add the data as a prop

Reply:
I'll address this change in challenge_3 pr